tests/Unit/Platforms/PlatformTest.php

```diff
@@ -59,6 +60,12 @@ public function getDownloadUrl(Application $application, Build $build, \Illumina
             {
                 return null;
             }
+
+            public function getPackage(UploadedFile $file): ?string
+            {
+                // TODO: Implement getPackage() method.
+                return null;
+            }
```

Comment:
What's the purpose of this one?

Reply:
To not break the tests, the compiler complained about a method not being implemented

Reply:
I could remove the TODO tho

Reply:
Done in https://github.com/everli/uppy/pull/43/commits/99a65e8219a85524d06876504fafbeba4a61e3ac